### PR TITLE
#167896277 Setup Babel for Project

### PR DESCRIPTION
**Pull request by 0xciph3r**
### What does this PR do?
- Sets up Babel for the project

### Description of Task Taken to Complete?
- Install  @babel/core @babel/cli @babel/preset-env @babel/node.
- Configure a .babelrc file
- Configure npm scripts to use  babel 

### How should this be manually tested?
- Not Applicable

### Any Background Context you want to Provide?
- None

### What are the relevant Pivotal Tracker Stories?
[167896277](https://www.pivotaltracker.com/story/show/167896277)

### Author
- Amadi Justice Chinonso

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+ "presets": ["@babel/preset-env"]
+}
```
Comment (Daymorelah) [inline .babelrc]: Add a new line here.

File: .gitignore [hunk 2]
```diff
@@ -54,6 +54,7 @@ typings/
 
 # Optional npm cache directory
 .npm
+package-lock.json
```
Comment (Daymorelah) [inline .gitignore:58]: Why is package-lock in .gitognore?.

**Review by Daymorelah — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 2 above)
Comment (0xciph3r) [reply .gitignore:58]: I will remove this

File: .babelrc (hunk 1 above)
Comment (0xciph3r) [reply .babelrc]: Okay I will do that

**Review by ChiamakaObitube — COMMENTED** [review]
@NonsoAmadi10 Great job here. The Babel configuration seems okay for the project.

**Comment by 0xciph3r** [comment]
> @NonsoAmadi10 Great job here. The Babel configuration seems okay for the prject.

Thank you for this

**Review by Daymorelah — APPROVED** [review]

**Pull request merged**